### DISCUSSION TITLE
Show Originals in Peepr

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#173] [Suggestion Box]
Show Originals is pretty useful for catching up on any important selfposts you might have missed from a friend, so it'd be nice to be able to use it in Peepr.

Truthfully, though, a universal toggle doesn't really feel completely right to me. Like the way the tagviewer tool is implemented on the site (as opposed to how activity filters are), filtering a blog page down to selfposts seems more like a specific action to me than a replacement effect. At least, that feels more natively Tumblr to me, and this webextension tends to stick to that rule well, intentionally or otherwise. Adding a button to the peepr header might not be worth it, though - this is pretty minor.

**Comment by aliadnan1984pk** [comment]
I also would love the ability to see originals in  peepr (in-dashboard blog viewer) . and best would be to have a checkbox option in "show originals" to toggle this for peepr  blog view
